### PR TITLE
Fix src bug.

### PR DESCRIPTION
**Pull request by evilebottnawi**
/cc @davidtheclark @jeddy3

### CODE REVIEW

**Comment by davidtheclark** [comment]
Should this be marked in the changelog?

**Comment by evilebottnawi** [comment]
@davidtheclark yep

**Comment by evilebottnawi** [comment]
@davidtheclark good?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Head
 
 - Fixed: `selector-type-no-unknown` now ignores nested selectors.
+- Fixed: `declaration-block-no-ignored-properties` now not ignore `src` properties, places out of `fontface` at-rules.
```
Comment (jeddy3) [inline CHANGELOG.md]: - Fixed: `declaration-block-no-duplicate-properties` now ignores `src` outside of `@font-face`.

**Comment by davidtheclark** [comment]
Sorry, I had a closer look. This needs to merge in to v7, right? Because it's a breaking change?

**Comment by davidtheclark** [comment]
I dunno, maybe I'm misunderstanding. I have to go, though, so you two can make the final call!

**Comment by jeddy3** [comment]
> I dunno, maybe I'm misunderstanding. I have to go, though, so you two can make the final call!

Oh jeez, you're right. I totally thought this was doing the opposite.

@evilebottnawi Why are we doing this? Doesn't `src` only appear in `@font-faces`?

**Comment by evilebottnawi** [comment]
@jeddy3 yes

**Comment by jeddy3** [comment]
Ok, so why this change?

**Comment by evilebottnawi** [comment]
@jeddy3 custom properties use `src` for other use case.

**Comment by evilebottnawi** [comment]
@jeddy3  throw postcss-plugins

**Comment by jeddy3** [comment]
> @jeddy3 throw postcss-plugins

Got an example? I'm having a bit of trouble grokking this.

**Comment by evilebottnawi** [comment]
@jeddy3 hm, I just thought that it could be used by someone in the future, if you think otherwise, we can close this issue.

**Pull request closed without merging**